docs/core/misc/layout-lifecycle.md

```diff
@@ -0,0 +1,196 @@
+# UI Layout Lifecycle
```

Comment:
IIRC you mentioned that this is now outdated, though it doesn't seems so on the first look.